### PR TITLE
[Task #1925670] Support open azure storage explorer for storage account

### PR DESCRIPTION
**Pull request by Flanker32**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Support open azure storage explorer for storage account
- Add storage explorer path in Azure settings panel
- Add open azure storage explorer for storage node

Does this close any currently open issues?
------------------------------------------
[AB#1925670](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1925670)

Any relevant logs, screenshots, error output, etc.?
-------------------------------------
N/A

Any other comments?
-------------------
N/A

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/resources/bundles/com/microsoft/azure/toolkit/operation.properties [hunk 1]
```diff
@@ -306,6 +306,7 @@ sqlserver.refresh_properties_view.server=refresh properties view of SQLServer({0
 
 storage.open_portal.account=open configuration page of storage account({0}) in portal
 storage.open_storage_explorer.account=open Storage Explorer of storage account({0})
+storage.open_azure_storage_explorer.account=open Azure Storage Explorer of storage account({0})
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/resources/bundles/com/microsoft/azure/toolkit/operation.properties:309]: unify storage account to `Storage account`

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java [hunk 2]
```diff
@@ -36,12 +38,20 @@ public void registerActions(AzureActionManager am) {
         final Action<StorageAccount> action = new Action<>(open, openView);
         action.setShortcuts(am.getIDEDefaultShortcuts().view());
         am.registerAction(OPEN_STORAGE_EXPLORER, action);
+
+        final Consumer<StorageAccount> openAzureStorageExplorer = resource -> new OpenAzureStorageExplorerAction().openResource(resource);
+        final ActionView.Builder openAzureStorageExplorerView = new ActionView.Builder("Open Azure Storage Explorer")
+                .title(s -> Optional.ofNullable(s).map(r -> title("storage.open_azure_storage_explorer.account", ((StorageAccount) r).name())).orElse(null))
+                .enabled(s -> s instanceof StorageAccount);
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java:40]: add ` && s.isConnected()`

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/AzurePanel.java [hunk 3]
```diff
@@ -250,5 +259,14 @@ public void reset() {
 
     private void createUIComponents() {
         this.funcCoreToolsPath = new FunctionCoreToolsCombobox(project, false);
+        this.txtStorageExplorer = new AzureFileInput() {
+            @Override
+            public boolean needValidation() {
+                return true;
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/AzurePanel.java:270]: Check shall we remove the overwrite as we added the validators already

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java [hunk 4]
```diff
@@ -65,4 +86,8 @@ public void registerGroups(AzureActionManager am) {
     public int getOrder() {
         return INITIALIZE_ORDER;
     }
+
+    public static void copyContentToClipboard(final String content) {
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/StorageActionsContributor.java:97]: Check duplicated for this method

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/action/explorer/LinuxOpenAzureStorageExplorerHandler.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+import java.io.IOException;
+
+public class LinuxOpenAzureStorageExplorerHandler extends AbstractAzureStorageExplorerHandler {
+    @Override
+    protected String getStorageExplorerExecutableFromOS() {
+        return null;
+    }
+
+    @Override
+    protected void launchStorageExplorer(String explorer, String storageUrl) {
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/action/explorer/LinuxOpenAzureStorageExplorerHandler.java:29]: Add `@AzureOperation`

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/action/explorer/AbstractAzureStorageExplorerHandler.java [hunk 6]
```diff
@@ -87,8 +92,9 @@ protected String getStorageExplorerExecutable() {
                 openResource(storageAccount);
             }
         };
-        final Action<Void> browserAction = new Action<>(browserConsumer, new ActionView.Builder("Set Path for Storage Explorer"));
-        return new Action[]{downloadAction, browserAction};
+        final Action<Void> browserAction = new Action<>(browserConsumer, new ActionView.Builder("Browser"));
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-storage-lib/src/main/java/com/microsoft/azure/toolkit/ide/storage/action/explorer/AbstractAzureStorageExplorerHandler.java:95]: `Browser` -> `Configure`

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**